autoload/ale/cursor.vim

```diff
@@ -136,8 +136,13 @@ function! s:ShowCursorDetailForItem(loc, options) abort
 
     let s:last_detailed_line = line('.')
     let l:message = get(a:loc, 'detail', a:loc.text)
-    let l:lines = split(l:message, "\n")
-    call ale#preview#Show(l:lines, {'stay_here': l:stay_here})
+
+    if g:ale_set_balloons == 1 && has('balloon_eval')
```

Comment:
Popups are more general and separate from the balloon feature. The feature check needs to check for `textprop` instead of `balloon_eval`.

There should probably also be a separate option, since at least I want popups, but no balloons as I find the mouse hover feature rather irritating.